lib/Compiler.js

```diff
@@ -232,7 +232,8 @@ Compiler.prototype.emitAssets = function(compilation, callback) {
 	function emitFiles(err) {
 		if(err) return callback(err);
 
-		require("async").forEach(Object.keys(compilation.assets), function(file, callback) {
+		var async = require("async");
```

Comment:
@sokra what name should be preferred for the `var`? `asyncLib`? Might be worth it to make a PR rewriting all uses of `async` as an identifier.